portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 Google LLC
```

Comment:
Big diffs are hard to review. A few reasons:

1. It's hard for the reader to understand everything.
2. It's hard for the implementor since they write a bunch of code, and then if the reviewer leaves significant feedback the implementor has to redo everything.

In the future, I would treat this CL as severak:

CL#1: Add empty CommentsServlet.java files with stub methods that throw exceptions.
CL#2: Implement the Get method.
CL#3: Implement the Post method
CL#4: Integrate the servlet w/ javascript and fix any servlet bugs that are found during the integration.

This makes it easier since I could leave the feedback of "Use a Comment object instead of strings" to save you the work of implement everything with those types. 

Let me know if you want to talk more about any of this :)

Reply:
Gotcha, sounds good. I think I struggled a bit this week trying to split up the commits/PRs, since each step of the tutorial had you change doGet/ doPost in different ways so I didn't know when a part was a part of a finished product that I could push, if that makes sense.

That being said, this diff was definitely far too big so I will be more aware of that in the future!